CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
```

Comment:
In the past we've included ### Added / ### Changed / etc so there are places to add changelog changes for the unreleased version.

Reply:
Good point, I'll do that in the future